tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -133,6 +133,24 @@ def pow(inp1: Field[[IDim], float64]) -> Field[[IDim], float64]:
     assert np.allclose(a.array() ** 2, b)
 
 
+def test_xor(fieldview_backend):
+    if fieldview_backend == gtfn_cpu.run_gtfn:
+        pytest.xfail("gtfn does not yet support math builtins")
+
```

Comment:
```suggestion
```
it does support math builtins. does it break?

Reply:
Can you also try to remove it from the other math builtin tests? Could be in a separate PR if you don't want to pollute this one.